x/auth/tx/builder.go

```diff
@@ -423,6 +423,14 @@ func (w *wrapper) GetProtoTx() *tx.Tx {
 	return w.tx
 }
 
+func (w *wrapper) GetRawTx() *tx.TxRaw {
+	return &tx.TxRaw{
+		BodyBytes:     w.bodyBz,
+		AuthInfoBytes: w.authInfoBz,
+		Signatures:    w.tx.Signatures,
```

Comment:
is there any chance `w.tx` is nil? Asking as that could yield a nil pointer exception

Reply:
In theory it should not be possible, because the constructor is internal

Reply:
We should be safe and add a nil-check then